data-tool/flows/tombstone/tombstone_queries.py

```diff
@@ -720,6 +720,27 @@ def get_filing_comments_query(corp_num):
     return query
 
 
+def get_in_dissolution_query(corp_num, threshold_timestamp):
+    query = f"""
+    select
+        cs.corp_num         as cs_corp_num,
+        cs.state_type_cd    as cs_state_type_cd,
+        e.event_id          as e_event_id,
+        e.event_type_cd     as e_event_type_cd,
+        to_char(
+            e.trigger_dts::timestamp at time zone 'UTC', 'YYYY-MM-DD HH24:MI:SSTZH:TZM'
+        )                   as e_trigger_dts_str
+    from corp_state cs
+    join event e on e.event_id = cs.start_event_id
+    where 1 = 1
+    and cs.corp_num = '{corp_num}'
+    and cs.end_event_id is null
+    and cs.state_type_cd in ('D1F', 'D2F', 'D1T', 'D2T')
+    and e.trigger_dts > '{threshold_timestamp}' :: timestamptz
```

Comment:
so i'm assuming we have corp state entries in D1T or D2T where the trigger date is before the threshold timestamp?  and if so, we have confirmed with Dylan what we're doing is ok?

Reply:
The decision log suggests that the current state for involuntary dissolution needs to be brought over. So I use this threshold to avoid duplicate processing in both LEAR and COLIN. But it depends on how we fill the gap between COLIN shutdown and migration dry run, which I still have no idea. Any thoughts?

Reply:
As per our discussion, I think we can remove the threshold timestamp.  The scenario where LEAR and COLIN have duplicate involuntary dissolution processing shouldn't happen.  It would mean our COLIN extract is not the latest data once COLIN data is frozen for corps.